### PR TITLE
drainer: add read-timeout configuration explanation

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11674
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lichunzhu — APPROVED** [review]
LGTM

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 1]
```diff
@@ -318,6 +318,7 @@ The following configuration items are related to connection to downstream databa
 * `port`: If this item is not set, TiDB Binlog tries to check the `MYSQL_PORT` environment variable which is `3306` by default.
 * `user`: If this item is not set, TiDB Binlog tries to check the `MYSQL_USER` environment variable which is `root` by default.
 * `password`: If this item is not set, TiDB Binlog tries to check the `MYSQL_PSWD` environment variable which is `""` by default.
+* `read-timeout`: Specifies the read timeout of database connections downstream. The default value is `"1m"`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```
Comment (shichun-0415) [inline tidb-binlog/tidb-binlog-configuration-file.md:321]:
```suggestion
* `read-timeout`: Specifies the I/O read timeout of database connections downstream. The default value is `1m`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 2]
```diff
@@ -318,6 +318,7 @@ The following configuration items are related to connection to downstream databa
 * `port`: If this item is not set, TiDB Binlog tries to check the `MYSQL_PORT` environment variable which is `3306` by default.
 * `user`: If this item is not set, TiDB Binlog tries to check the `MYSQL_USER` environment variable which is `root` by default.
 * `password`: If this item is not set, TiDB Binlog tries to check the `MYSQL_PSWD` environment variable which is `""` by default.
+* `read-timeout`: Specifies the I/O read timeout of database connections downstream. The default value is `1m`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```
Comment (Oreoxmt) [inline tidb-binlog/tidb-binlog-configuration-file.md:321]:
```suggestion
* `read-timeout`: Specifies the I/O read timeout of the downstream database connection. The default value is `1m`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 3]
```diff
@@ -318,6 +318,7 @@ The following configuration items are related to connection to downstream databa
 * `port`: If this item is not set, TiDB Binlog tries to check the `MYSQL_PORT` environment variable which is `3306` by default.
 * `user`: If this item is not set, TiDB Binlog tries to check the `MYSQL_USER` environment variable which is `root` by default.
 * `password`: If this item is not set, TiDB Binlog tries to check the `MYSQL_PSWD` environment variable which is `""` by default.
+* `read-timeout`: Specifies the I/O read timeout of the downstream database connection. The default value is `1m`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```
Comment (shichun-0415) [inline tidb-binlog/tidb-binlog-configuration-file.md:321]:
```suggestion
* `read-timeout`: Specifies the I/O read timeout of the downstream database connection. The default value is `1m`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value. 
```

**Pull request merged**